examples/nextjs-next-with-typescript/next.config.js

```diff
@@ -1,3 +1 @@
-const withTypescript = require('@zeit/next-typescript');
-
-module.exports = withTypescript();
+module.exports = {};
```

Comment:
Do we need this file?

Reply:
doesn't look like it, I just aligned with the basic next example. Will remove

Reply:
@oliviertassinari Ok, I see now. If I remove this file it tries to pick up the `next.config.js` from the root of the project.